templates/client_specific_config.erb

```diff
@@ -7,6 +7,9 @@ iroute-ipv6 <%= iroute_ipv6 %>
 <% if ifconfig = @ifconfig -%>
 ifconfig-push <%= ifconfig %>
 <% end -%>
+<% if ifconfig_ipv6 = @ifconfig_ipv6 -%>
```

Comment:
what is the purpose of this condition? Can't we just do:

```erb
<% if ifconfig_ipv6 -%>
```

Reply:
Can be, I just followed the pattern from `ifconfig` parameter.